doc_config/yard/setup.rb

```diff
@@ -13,8 +13,7 @@ def html_syntax_highlight_ruby(source)
         private
 
         def highlight_with_pygments(language, source)
```

Comment:
Do we need to support languages other than ruby?

Reply:
I don't think so — I think this is fine for now.

Reply:
I already added the support on later commit, should i revert back to using `ruby`?

Reply:
Oh, right, that makes sense. That's okay, what you did is fine too. :)